.github/CONTRIBUTING.md

```diff
@@ -23,21 +23,43 @@ This document contains guidelines specific to Rhino.
 6. Build package<br>
 `devtools::build()` or `pkgbuild::build()`
 
+## App Push Test
+
+Rhino creates a `.github/workflows/rhino-test.yml` file on `rhino::init()`.
+This GitHub Actions workflow automatically runs all linters and tests
+once a project is pushed to GitHub.
+To test that it works correctly we have the [App Push Test](https://github.com/Appsilon/rhino/actions/workflows/app-push-test.yml) workflow.
+
+The Push Test initializes a fresh Rhino application and pushes it to the `bot/app-push-test` branch.
+The Rhino Test of this application then runs and its results can be viewed
+in the [list of workflow runs](https://github.com/Appsilon/rhino/actions/workflows/rhino-test.yml).
+The App Push Test is triggered automatically on pushes to `main` and can also be triggered manually.
+
 ## Release process
 
 ### Preparation
 
-1. Announce the planned release on `#proj-rhino`
-(approximate date and scope).
-2. Bump the package version in `DESCRIPTION` according to [SemVer](https://semver.org/).
-Drop the development version (last component, e.g. `.9001`).
-3. Update `NEWS.md`.
-    1. Replace the `(development version)` with `X.Y.Z` in the header.
-    Do add a link to GitHub releases yet - the link won't work and will fail CRAN checks.
-    2. Edit the list of changes to make it useful and understandable for our users.
-    See [keep a changelog](https://keepachangelog.com/) for some guidelines.
-4. Submit the changes in a pull request titled "Release X.Y.Z".
-Get it approved and merged.
+1. Announce the planned release on `#proj-rhino` (approximate date and scope).
+2. Plan promotion (social media, blog post, ...).
+Coordinate efforts with the marketing team.
+3. Ensure that [App Push Test](#app-push-test) passes.
+4. Upgrade [Rhino Showcase](https://github.com/Appsilon/rhino-showcase).
+    1. Create a task to track progress.
+    2. Test upgrade by installing Rhino from the current `main` branch.
+    3. Continue the release process.
+    The upgrade can be completed and the task closed once the package is accepted to CRAN.
+5. Prepare the package for release.
+    1. Create a `release-X.Y.Z` branch from `main`.
```

Comment:
I wonder if we should mention here the differentiation between `release` and `release-candidate`. What do you think?

Reply:
OK, I see you did it below, you can ignore this comment